manuscript/060_Statement_First_Programming.md

```diff
@@ -63,7 +63,7 @@ public class CalculatorSpecification
 }
 ```
 
-Now, imagine that you are writing this Statement post-factum as a�unit test in an environment that has, let's say, more than thirty Statements -- you have written the code, now you are just creating a�test after test �to ensure" (as you see, this is not my favorite reason for writing unit tests) the code works. Code, test -- pass, test -- pass, test -- pass. You almost always evaluate your code against the whole Specification, since it is usually easier than selecting what to evaluate each time, plus, you get more confidence this way that you did not break by mistake something that is already working. So, this is really: Code, Test -- all pass, test -- all pass, test -- all pass... Hopefully, you use some kind of snippets mechanism for creating new Statements, but if not (and many do not actually do this), once in a�while, you do something like this:
+Now, imagine that you are writing this Statement post-factum as a unit test in an environment that has, let's say, more than thirty Statements. You have written the code and now you are just creating test after test “to ensure" the code works. Code, test -- pass, test -- pass, test -- pass. Usually you evaluate your code against the whole Specification, since it is easier than selecting what to evaluate each time. Besides you get more confidence this way that you don't make a mistake and break something that is already working. So, this is really: Code, Test -- all pass, test -- all pass, test -- all pass... Hopefully, you use some kind of snippets mechanism for creating new Statements, otherwise you might write something like this once in a while:
```

Comment:
When the "(as you see, this is not my favorite reason for writing unit tests)" is left out, the  quotation marks around "to ensure" add no value in my mind. Let's remove them. By the way, now I see that I should have written this as a story with me as the main character, not the poor reader :-D.